scripts/smithy.lic

```diff
@@ -0,0 +1,137 @@
+=begin
+
+  Smithy service helper script
+
+  This script will request multiple services from the smithy. It tries to be
+  strict about what it will accept for input and to verify that a blacksmith
+  NPC is present before requesting services.
+
+  Usage:
+
+  ;smithy <count> <crit, damage, or sighting>
+
+  author: Kragdruk
+  version: 1.0
+  tags: util, ccf, duskruin, festival, merchant
+
+=end
+
+module Smithy
+  SERVICE_TYPES = %w[critical damage sighting]
+
+  def self.determine_service_type_from(arg)
+    SERVICE_TYPES.find { |t| t.start_with? arg.downcase }
+  end
+
+  BLACKSMITH_DESCRIPTIONS = [
```

Comment:
I set it up like this in case the DR and CCF blacksmiths had different descriptions, but I checked my logs and they are the exact same. The cost of abstraction is minimal here and I also abused it for some simple testing, so figured I'd keep it like this even though there is only one entry in the list.